libwarpaffine/utilities.cpp

```diff
@@ -438,6 +438,46 @@ int Utilities::StrcmpCaseInsensitive(const char* a, const char* b)
 #endif
 }
 
+libCZI::GUID Utilities::GenerateGuid()
+{
+#if defined(_WIN32)
```

Comment:
I'd suggest preferring to check the define LIBWARPAFFINE_WIN32_ENVIRONMENT here. It is used to control whether Windows.h is included here or not (line 18).